src/main/java/com/epam/test_generator/entities/Case.java

```diff
@@ -39,9 +40,8 @@
     private Set<Tag> tags;
 
     public Case() {
-        SimpleDateFormat formatter = new SimpleDateFormat("MM/dd/yyyy");
-        creationDate = formatter.format(Calendar.getInstance().getTime());
-        updateDate = formatter.format(Calendar.getInstance().getTime());
+        creationDate = Calendar.getInstance().getTime();
```

Comment:
I'd recommend excluding time to a local variable because if GC is initiated between getTime executions and you'll get different values in creationDate and updateDate.